README.md

```diff
@@ -0,0 +1,149 @@
+# UMU-Protonfixes
+
+This project acts as a stopgap for fixing games that don't work out of the box on Proton. It implements game-specific patches, such as installing Winetricks verbs.
+
+It is the heart of `UMU-Proton` and `GE-Proton`.
+
+## Start parameter fixes
+
+The [default fix file for Steam](gamefixes-steam/default.py) implements a feature to install Winetricks / Protontricks verbs via startup parameters. Just add `-pf_tricks=<verb>` to it. You can install multiple verbs by adding the parameter more than once.
+
+```bash
+-pf_tricks=xliveless -pf_tricks=d3dcompiler_47
+```
+
+You can also set [dxvk options](https://github.com/doitsujin/dxvk/wiki/Configuration).
+
+```bash
+-pf_dxvk_set=dxgi.maxFrameRate=40 -pf_dxvk_set=d3d9.maxFrameRate=40 
+```
+
+The order is not important and the arguments are not passed on to the game.
+
+## Local fixes
+
+The easiest way to implement a new fix or modify an existing one, is to use local fixes.
+
+First of all, you should find the Steam-ID of the game (right click the game in your Steam library -> 'Properties...' -> 'Updates' -> 'App ID:') and use it as the filename, with a `.py` extension.
+
+Example `Day of Defeat: Source` (AppID 300):
+
+> ~/.config/protonfixes/localfixes/300.py
+
+Another type of fix is one that runs before each game. This allows you to set defaults for all games.
+
+> ~/.config/protonfixes/localfixes/default.py
+
+Note that local fixes override the included "global" fixes. So using an existing fix might be a good starting point for modifications, as they will not be executed.
+
+For example, overriding the default fix will disable [the parameter parsing for Steam games](#start-parameter-fixes). This can be restored by copying [the global default file](gamefixes-steam/default.py) and applying your changes accordingly.
+
+## Non-Steam games
+
+As we support platforms other than Steam, these fixes can also be used by other launchers, such as Lutris, Heroic, Legendary or Bottles.
+
+The current implementation requires entries in a special database that looks up the `UMU-ID`, which consists of the prefix `umu-` and in most cases the SteamID (if the game exists on that platform).
+
+https://github.com/Open-Wine-Components/umu-database
+
+## Config file
+
+There is a config file located at:
+
+> ~/.config/protonfixes/config.ini
+
+It's not widely used at the moment, but you can configure some aspects.
```

Comment:
Agreed. I doubt this feature is widely used too, so we should definitely consider deprecating this feature along with the GDrive functionality.

Reply:
Or we can add more config parameters and actually use that feature. Not sure yet.